imports/plugins/included/default-theme/client/styles/staticPages/staticPages.less

```diff
@@ -0,0 +1,69 @@
+#static-pages-wrapper {
+  padding-left: 250px;
+  transition: all 0.4s ease 0s;
+  overflow-x: scroll;
+  height: 75vh;
+}
+
+#sidebar-wrapper {
```

Comment:
**Nitpick**
I would recommend not using too many negative margins in your application. Flexbox model can come in handy for you